tgui/packages/tgui/interfaces/NuclearBomb.js

```diff
@@ -0,0 +1,93 @@
+import { useBackend } from '../backend';
+import { Box, Button, LabeledList, Section } from '../components';
+import { Window } from '../layouts';
+
+export const NuclearBomb = (props, context) => {
+  const { act, data } = useBackend(context);
+
+  if (!data.extended) {
+    return (
+      <Window>
+        <Window.Content>
+          <Section title="Deployment">
+            <Button
+              icon="power-off"
```

Comment:
```suggestion
              icon="exclamation-triangle"
```
Makes more sense than a power button in my opinion